validator/pom.xml

```diff
@@ -106,7 +105,7 @@
         </dependency>
 
         <dependency>
-            <groupId>com.github.tomakehurst</groupId>
+            <groupId>org.wiremock</groupId>
```

Comment:
This seems unrelated. Why is this dependency changing?

Reply:
A warning was indicating that the groupId had changed for wiremock so I figured it would be better to use the new one